src/optional.rs

```diff
@@ -8,6 +10,7 @@ pub enum Optional<T> {
     Value(T),
     Absent,
 }
+use Optional::*;
```

Comment:
It could be `use Optional::{Value, Absent}`. But on the other hand the definition is right up there...